addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -833,40 +853,23 @@ void shouldGenerateValidJsonPlusReport() throws Exception {
         assertThat(site.getJSONObject(0).getString("@host"), is(equalTo("example.com")));
         assertThat(site.getJSONObject(0).getString("@port"), is(equalTo("80")));
         assertThat(site.getJSONObject(0).getString("@ssl"), is(equalTo("false")));
-        checkAlert(site.getJSONObject(0));
+        checkAlert(site.getJSONObject(0), true);
 
         JSONArray alerts = site.getJSONObject(0).getJSONArray("alerts");
         assertThat(alerts.size(), is(equalTo(1)));
-        checkAlert(site.getJSONObject(0));
+        checkAlert(site.getJSONObject(0), true);
         // tags are not included in non plus report
         JSONArray tags = alerts.getJSONObject(0).getJSONArray("tags");
         assertThat(tags.size(), is(equalTo(1)));
         assertThat(tags.getJSONObject(0).getString("tag"), is(equalTo("tagkey")));
         assertThat(tags.getJSONObject(0).getString("link"), is(equalTo("tagvalue")));
     }
 
-    @Test
-    void shouldGenerateValidXmlReport() throws Exception {
-        // Given
-        Template template = getTemplateFromYamlFile("traditional-xml");
-        String fileName = "basic-traditional-xml";
-        File f = File.createTempFile(fileName, template.getExtension());
-        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
-        dbf.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
-        DocumentBuilder db = dbf.newDocumentBuilder();
+    void checkXMLAlert(Document doc, Boolean isXMLPlus) {
```

Comment:
checkXmlPlus

Reply:
the Boolean variable name or the function name ? Actually this functions checks for both Xml and Xml-Plus. If isXMLPlus is true then it check for xml-plus else for xml.

Reply:
Both should follow camelCaps